rfcs/0190-queue-change-task-run-priority.md

```diff
@@ -0,0 +1,55 @@
+# RFC 0190 - Queue change task run priority
+* Comments: [#0190](https://github.com/taskcluster/taskcluster-rfcs/pull/190)
+* Proposed by: @lotas
+
+## Summary
+
+Since we refactored queue internals it became possible to see what tasks are pending for a particular
+worker pool / task queue.
+
+This RFC proposes new API method that would allow to change the priority of the existing task run.
+
+## Motivation
```

Comment:
We might want to expand on this motivation section, and list actual concrete intended use cases, to make it clearer what is/isn't in scope.